nexus/src/external_api/http_entrypoints.rs

```diff
@@ -490,26 +490,18 @@ async fn silo_list(
 ) -> Result<HttpResponseOk<ResultsPage<Silo>>, HttpError> {
     let apictx = rqctx.context();
     let nexus = &apictx.nexus;
+    let query = query_params.into_inner();
```

Comment:
nit: a downside of putting this stuff outside the `handler` async block is that the request timers won't account for the time required to do this.  Hopefully it's not much but that's generally why I try to keep almost everything inside that block.

Reply:
Cool, I didn't realize that. I'm happy to move stuff back into the handler.